scraper.rb

```diff
@@ -25,20 +28,12 @@ def scrape_list(url)
   noko = noko_for(url)
   puts url.to_s.yellow
```

Comment:
This is inherited from the existing version, but Morph doesn't play well with `colorize`.